src/components/Monaco/DiffEditor.tsx

```diff
@@ -0,0 +1,39 @@
+import { Editor, DiffEditor as MonacoDiffEditor } from '@monaco-editor/react'
+import { editor } from 'monaco-editor'
+import { ComponentProps } from 'react'
+
+import { useTheme } from '@/hooks/useTheme'
+
+import { DEFAULT_OPTIONS } from './defaultOptions'
+const DEFAULT_DIFF_OPTIONS: editor.IDiffEditorOptions = {
+  ...DEFAULT_OPTIONS,
+  readOnly: true,
+  renderOverviewRuler: false,
+  lineNumbers: 'off',
+}
+
+interface Props extends ComponentProps<typeof Editor> {
+  original?: string
+}
+
+export function DiffEditor({ original, value, language, options }: Props) {
+  const theme = useTheme()
+  return (
+    <MonacoDiffEditor
```

Comment:
Shouldn't it also have the line wrap button? Currently, it does not, which leads to layout shift when switching between content and diff
<img width="368" alt="image" src="https://github.com/user-attachments/assets/466a3aab-b822-4e56-ac85-6e7f668d49a2" />